src/utils/flow.js

```diff
@@ -43,6 +43,7 @@ export function toValidTabSlug(tabSlug: any): TabSlug | null {
   const coercedTabSlug = (tabSlug: TabSlug);
   switch (coercedTabSlug) {
     case 'calltree':
+    case 'functionTable':
```

Comment:
The other panels use the format `function-table`.

Reply:
You're right. It's updated (along side the tab slug name locale property).